app/Packages/Quiz/Question/Domain/Model/Question.php

```diff
@@ -35,10 +38,33 @@ public function __construct(
          *     cascade={"persist", "remove"},
          * )
          */
-        private ?ArrayCollection $alternativeQuestions = null
+        private ?Collection $alternativeQuestions = null
     )
     {
-        $this->subjects = new ArrayCollection();
         $this->alternativeQuestions = new ArrayCollection();
     }
+
+    public function addAlternativeQuestions(Collection $alternativeQuestions): void

```

Comment:
Acho que isso vai acabar sendo inútil, exceto se eu adicionar uma coluna com as possíveis alternativas pra cada questão.
Não sei muito bem o que fazer ainda, mas sei que na hora de lançar a nota de cada Quiz vai ser doloroso. Acho que vou tirar, por enquanto e esperar o problema chegar